src/models/window.rs

```diff
@@ -205,9 +214,9 @@ impl Window {
         }
         if self.floating() && self.floating.is_some() {
             let relative = self.normal + self.floating.unwrap();
-            relative.y() + self.margin
+            relative.y() + self.margin.clone().bottom()
         } else {
-            self.normal.y() + self.margin
+            self.normal.y() + self.margin.clone().bottom()
```

Comment:
another question if I may: why are you using `.bottom()` here and no `.top()`? Isn't the origin for determining window-positions the top-left screen corner?

Reply:
Now that you mention it, that’s probably right. This might be an artifact from when I was confused and had the HTML top/bottom/ left/right specs backwards.  

Should probably check over and see if the behaviour is what we’re expecting it to be.

Reply:
as I am just about to get a `MarginMultiplier` PR ready (just struggling to get the external command working) so I might include this..

Reply:
That sounds great! Thanks for reviewing that. I wonder if it's partially related to #248

Reply:
> I wonder if it's partially related to #248

Gonna look into this tomorrow, now it's bedtime. Gn8